src/test/java/org/opentripplanner/routing/algorithm/transferoptimization/model/OptimizedPathTailTest.java

```diff
@@ -60,6 +60,7 @@ class OptimizedPathTailTest implements RaptorTestConstants {
   private final OptimizedPathTail<TestTripSchedule> subject = new OptimizedPathTail<>(
     SLACK_PROVIDER,
     BasicPathTestCase.COST_CALCULATOR,
+    0,
```

Comment:
Make this a constant: ANY (if ignored by the test) or ITERATION_DEPATURE_TIME_ZERO if used.